pyproject.toml

```diff
@@ -151,7 +151,7 @@ known_first_party = ["migrations", "test"]
 
 [tool.black]
 line-length = 120
-target-version = ["py39"]
+target-version = ["py310"]
```

Comment:
Not sure if this should already be the target-version; will the code still work for py39?

Reply:
Yeah it should. mypy complained that I should update this, presumable because there are version checks in the code now. It's only for black, regardless.